relayer/src/lib.rs

```diff
@@ -173,6 +174,7 @@ async fn relay(
     // The pending transaction itself doesn't serialize well, but all the relevant information is
     // contained in the transaction hash. The client can reconstruct the pending transaction from
     // the hash using a particular provider.
+    event!(Level::INFO, "ETH H256: {}", *pending);
```

Comment:
```suggestion
    event!(Level::INFO, "Submitted ethereum transaction hash: {:?}", *pending);
```
The exact wording is not so important but I think searching for "transaction hash" should find these logs.

Also without the debug formatting it prints an abbreviated form:

```
0x8ca3…9922
```
instead of
```
0x3eac86d83469d6c1f95d4776b5fe276df1321c761e64311d534feef09339eb45
```
The abbreviated form would make it difficult or impossible to lookup in a blockchain explorer.

Reply:
FYI you can also use `{:x}` or `{:#x}` to print the full hex address without relying on `Debug`